examples/starburst_trino_NYCTaxi.ipynb

```diff
@@ -0,0 +1,552 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "619ca82d-2963-4586-bfab-d346e1ef0c24",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nycjanduration = nyc_filtered.mutate(rideminutes=nyc_filtered.tpep_dropoff_datetime.delta(nyc_filtered.tpep_pickup_datetime, \"minute\"))\n",
+    "nycjanduration[\"vendorid\",\"rideminutes\",\"trip_distance\"]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "75b1628a-d00d-4c68-adef-511a492866c9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nycjanduration[\"vendorid\",\"rideminutes\",\"trip_distance\"].head(3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "780d6f0e-e3ac-4c98-a8db-709bd6bdad5a",
+   "metadata": {},
+   "source": [
+    "Next up are some basic analytics and aggretions in Ibis  - let's get total revenue with sum(), longest trip with max(), and average trip duration with mean(). \n",
+    "Ibis is able to chain expressions similar to pandas. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ce3c154a-b367-41f8-962c-473433b07b86",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#some basic analytics - let's get total revenue, longest trip. \n",
+    "insights = nycjanduration.agg(\n",
+    "    [\n",
+    "        ibis._.count().name(\"total_trips\"),\n",
+    "        ibis._[\"total_amount\"].sum().name(\"total_revenue\"),\n",
+    "        ibis._[\"trip_distance\"].sum().name(\"total_distance_all\"),\n",
+    "        ibis._[\"rideminutes\"].max().name(\"longest\"),    \n",
+    "        ibis._[\"rideminutes\"].mean().round(2).name(\"average_ride\")\n",
+    "    ]\n",
+    ")\n",
+    "insights"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "957102d5-2ee7-485c-ae44-7c8e3817bfb9",
+   "metadata": {},
+   "source": [
+    "Wait, the longest trip seems a bit... lengthy... Note: we added a .round function to display the average ride more nicely. Let's check out the ride itself. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9086a984-f484-43eb-a774-e1db784ea3e4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nycjanduration.filter(nycjanduration[\"rideminutes\"] == 10030)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5c4e3e47-d180-412e-b3d4-46cabfbf8ea3",
+   "metadata": {},
+   "source": [
+    "7 day trip? looks like the trip distance is zero, we can decide to remove the row from future calculations of average\n",
+    "Let's remove the outliers and join with a lookup table to get more information about the \"where\" of our analytical datasets - zones."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "99a27e7f-f626-4388-8613-75001f5a9f38",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nycjanduration_new = (\n",
+    "    nycjanduration.filter(nycjanduration.trip_distance != 0.0)\n",
+    ")\n",
+    "nycjanduration_new\n",
```

Comment:
If the `nycjanduration` expression isn't going to be used after this, I would suggest just using the same variable name.  You avoid all the `_new` in the names, e.g.

```python
nycjanduration = nycjanduration.filter(_.trip_distance > 0)
```